src/fieldmap.c

```diff
@@ -365,43 +365,43 @@ union Block
     u16 value;
 };
 
-#define MapGridGetBorderTileAt(x, y) ({                        \
-    u16 block;                                                 \
-    s32 xprime;                                                \
-    s32 yprime;                                                \
-                                                               \
-    struct MapLayout *mapLayout = gMapHeader.mapLayout;              \
-                                                               \
-    xprime = x - 7;                                            \
-    xprime += 8 * mapLayout->unk18;                              \
-    xprime %= mapLayout->unk18;                                  \
-                                                               \
-    yprime = y - 7;                                            \
-    yprime += 8 * mapLayout->unk19;                              \
-    yprime %= mapLayout->unk19;                                  \
-                                                               \
-    block = mapLayout->border[xprime + yprime * mapLayout->unk18]; \
-    block |= 0xC00;                                            \
-    block;                                                     \
+#define MapGridGetBorderTileAt(x, y) ({                                  \
+    u16 block;                                                           \
+    s32 xprime;                                                          \
+    s32 yprime;                                                          \
+                                                                         \
+    struct MapLayout *mapLayout = gMapHeader.mapLayout;                  \
+                                                                         \
+    xprime = x - 7;                                                      \
+    xprime += 8 * mapLayout->borderWidth;                                \
+    xprime %= mapLayout->borderWidth;                                    \
+                                                                         \
+    yprime = y - 7;                                                      \
+    yprime += 8 * mapLayout->borderHeight;                               \
+    yprime %= mapLayout->borderHeight;                                   \
+                                                                         \
+    block = mapLayout->border[xprime + yprime * mapLayout->borderWidth]; \
+    block |= METATILE_COLLISION_MASK;                                    \
+    block;                                                               \
```

Comment:
you don't need this line. `block |= METATILE_COLLISION_MASK` is a valid expression. 
same issue below

Reply:
Ye I was just labeling the fields, that was already there. Can do ya one better, the whole second macro is unnecessary.